sdk/monitor/opentelemetry-exporter-azuremonitor/samples/traces/server.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# pylint: disable=import-error
+# pylint: disable=no-member
+# pylint: disable=no-name-in-module
+import requests
+from opentelemetry import trace
+from opentelemetry.ext.flask import FlaskInstrumentor
+from opentelemetry.ext.requests import RequestsInstrumentor
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
+
+import flask
+from opentelemetry.exporter.azuremonitor import AzureMonitorSpanExporter
+
+# The preferred tracer implementation must be set, as the opentelemetry-api
+# defines the interface with a no-op implementation.
+trace.set_tracer_provider(TracerProvider())
+tracer = trace.get_tracer(__name__)
+
+exporter = AzureMonitorSpanExporter(
+    connection_string="InstrumentationKey=<INSTRUMENTATION KEY HERE>"
```

Comment:
must encourage the of env variables instead of pluggable in samples. There have been multiple cases where secrets were accidentally pushed ;)
```suggestion
    connection_string = os.environ["AZURE_MONITOR_CONNECTION_STRING"]
```